WooCommerce/Classes/POS/ViewModels/ItemSelectorViewModel.swift

```diff
@@ -4,14 +4,30 @@ import protocol Yosemite.POSItem
 import protocol Yosemite.POSItemProvider
 
 final class ItemSelectorViewModel: ObservableObject {
```

Comment:
Unrelated nitpick:

I think calling this VM `ItemListViewModel` would be more appropriate and increase understanding. `ItemSelectorViewModel` is not only a selector but also an item loader, and a sole VM used by `ItemListView`

Reply:
Sure! Initially we used "selector" because design switched a few times between list and grid style, but since we're settled on list this fits better now. If we have a switcher in the future we can abstract this further. Updated on https://github.com/woocommerce/woocommerce-ios/pull/13289/commits/79152b8d92cd61aa3e23d81e99ca5ec282005663